modules/monitoring-configuring-external-alertmanagers.adoc

```diff
@@ -3,144 +3,116 @@
 
 .Prerequisites
 
+// tag::CPM[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+// end::CPM[]
+// tag::UWM[]
 ifndef::openshift-dedicated,openshift-rosa[]
-* *If you are configuring core {product-title} monitoring components in the `openshift-monitoring` project*:
-** You have access to the cluster as a user with the `cluster-admin` cluster role.
-** You have created the `cluster-monitoring-config` config map.
-* *If you are configuring components that monitor user-defined projects*:
-** You have access to the cluster as a user with the `cluster-admin` cluster role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
-** A cluster administrator has enabled monitoring for user-defined projects.
+* You have access to the cluster as a user with the `cluster-admin` cluster role or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+* A cluster administrator has enabled monitoring for user-defined projects.
 endif::openshift-dedicated,openshift-rosa[]
 ifdef::openshift-dedicated,openshift-rosa[]
 * You have access to the cluster as a user with the `dedicated-admin` role.
 * The `user-workload-monitoring-config` `ConfigMap` object exists. This object is created by default when the cluster is created.
 endif::openshift-dedicated,openshift-rosa[]
+// end::UWM[]
 * You have installed the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Edit the `ConfigMap` object.
-ifndef::openshift-dedicated,openshift-rosa[]
-** *To configure additional Alertmanagers for routing alerts from core {product-title} projects*:
-.. Edit the `cluster-monitoring-config` config map in the `openshift-monitoring` project:
+. Edit the `{configmap-name}` config map in the `{namespace-name}` project:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+$ oc -n {namespace-name} edit configmap {configmap-name}
 ----
 
-.. Add an `additionalAlertmanagerConfigs:` section under `data/config.yaml/prometheusK8s`.
-
-.. Add the configuration details for additional Alertmanagers in this section:
+. Add an `additionalAlertmanagerConfigs:` section with configuration details under 
+// tag::CPM[]
+`data/config.yaml/prometheusK8s`:
+// end::CPM[]
+// tag::UWM[]
+`data/config.yaml/<component>:`
+// end::UWM[]
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: cluster-monitoring-config
-  namespace: openshift-monitoring
+  name: {configmap-name}
+  namespace: {namespace-name}
 data:
   config.yaml: |
+# tag::CPM[]
     prometheusK8s:
+# end::CPM[]
+# tag::UWM[]
+    <component> # <2>
+# end::UWM[]
       additionalAlertmanagerConfigs:
-      - <alertmanager_specification>
+      - <alertmanager_specification> # <1>
```

Comment:
Is the callout sequence here correct? <2> before <1>?

Reply:
Oh yes, I made it this way because of the tagging. If it was the other way around, it would be a little more complicated to tag. I have not seen rules agains this so I took it as an advantage here :grinning: